src/explore-education-statistics-frontend/src/modules/permalink/PermalinkPage.tsx

```diff
@@ -141,12 +138,19 @@ const PermalinkPage: NextPage<Props> = ({ data }) => {
 export const getServerSideProps: GetServerSideProps<Props> = async ({
   query,
 }) => {
-  const { permalink } = query;
-  const data = await permalinkService.getPermalink(permalink as string);
+  const { newPermalinks, permalink } = query as Dictionary<string>;
+  // TO DO - EES-4259 remove `newPermalinks` and tidy up
+  let data;
```

Comment:
Similar to my other comment, think we should add a type param here so `data` isn't inferred as `any`